sway-core/src/concurrent_slab.rs

```diff
@@ -57,63 +52,38 @@ impl<T> ConcurrentSlab<T>
 where
     T: Clone,
 {
+    pub fn len(&self) -> usize {
+        let lock = self.inner.read().unwrap();
```

Comment:
Proposal: let's use `inner` instead of `lock` here as the variable name. It is more descriptive and also in all other methods it is called `inner`.